apps/amiapp_flutter/ios/Runner/AppDelegate.swift

```diff
@@ -21,11 +20,17 @@ import FirebaseCore
 
         Messaging.messaging().delegate = self
         
+        // TODO: Fix SDK initialization
```

Comment:
should we not just remove it? because its not going to be needed anymore?

Reply:
Yeah, you're right. I initially kept it thinking it would be replaced with `MessagingPush` initialization. But since it's already in a separate block, we can remove it entirely. I've removed it from the code.